src/common/csv_reader/csv_reader.cpp

```diff
@@ -318,6 +318,38 @@ std::unique_ptr<Value> CSVReader::getList(const DataType& dataType) {
         DataType(LIST, std::make_unique<DataType>(dataType)), std::move(listVal));
 }
 
+std::unique_ptr<Value> CSVReader::getTensor(const DataType& dataType) {
+    std::vector<std::unique_ptr<Value>> tensorVal;
```

Comment:
hmm, I don't follow changes here. Where is this function used?

Reply:
Removed this function